common/transport/http/src/rest_api_client.ts

```diff
@@ -142,6 +145,8 @@ export class RestApiClient {
     }
 
     request.end();
+
+    return request;
```

Comment:
>return request; [](start = 4, length = 15)

I'm really not a fan of leaking the request object. The whole purpose of the RestApiClient is to abstract it. 

If there really is a need to be able to abstract request, let's try to think about a way to surface this without leaking the node API layer.